lib/Onyx.js

```diff
@@ -1106,11 +1094,10 @@ function clear(keysToPreserve = []) {
                 notifyCollectionSubscribersOnNextTick(key, value);
             });
 
-            // Call clear() and make sure that the default key/values and the key/values from the parameter
-            // are preserved in storage. This makes sure to always leave storage in a state that contains
-            // all the default values and any additional values that we want to remain after the database is cleared.
-            return Storage.clear()
-                .then(() => Storage.multiSet([...defaultKeyValuePairs, ...keyValuesToPreserve]));
+            const defaultKeyValuePairs = _.pairs(_.omit(defaultKeyStates, ...keysToPreserve));
```

Comment:
Unnecessary spread:

```suggestion
            const defaultKeyValuePairs = _.pairs(_.omit(defaultKeyStates, keysToPreserve));
```

<img width="374" alt="image" src="https://user-images.githubusercontent.com/47436092/222606585-35a726f9-39f7-4393-a470-9f2f1ff5bd6e.png">

Reply:
Fixed, thanks